student.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Student < Person
+  attr_reader :classroom
+
+  def initialize(id, classroom, age: 0, parent_permission: true, name: 'Unknown')
+    super(id, name, age, parent_permission)
```

Comment:
- [ ] As I mentioned in the previous comment that only `parent_permission` and `name` can have default values, kindly ensure that you also update the above `initialize` method in order to correctly match the project requirements. After you have used the `Random` method to generate `id`s you don't need to list it again as `parameter`. I should look similar to this one below:
```ruby
def initialize(age, classroom, name = "Unknown", parent_permission: true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end
```